Sources/SwiftGen/Commander/ParserCLICommands.swift

```diff
@@ -9,38 +9,42 @@ import PathKit
 import StencilSwiftKit
 import SwiftGenKit
 
-// deprecated option
-private let deprecatedTemplateNameOption = Option<String>(
-  "template",
-  default: "",
-  flag: "t",
-  description: """
-    DEPRECATED, use `--templateName` instead
-    """
-)
+extension ParserCLI {
+  private enum CLIOption {
+    // deprecated option
+    static let deprecatedTemplateName = Option<String>(
+      "template",
+      default: "",
+      flag: "t",
+      description: """
+        DEPRECATED, use `--templateName` instead
+        """
+    )
 
-private let templateNameOption = Option<String>(
-  "templateName",
-  default: "",
-  flag: "n",
-  description: """
-    The name of the template to use for code generation. \
-    See `swiftgen templates list` for a list of available names
-    """
-)
+    static let templateName = Option<String>(
+      "templateName",
+      default: "",
+      flag: "n",
+      description: """
+        The name of the template to use for code generation. \
+        See `swiftgen templates list` for a list of available names
+        """
+    )
 
-private let templatePathOption = Option<String>(
-  "templatePath",
-  default: "",
-  flag: "p",
-  description: "The path of the template to use for code generation."
-)
+    static let templatePath = Option<String>(
+      "templatePath",
+      default: "",
+      flag: "p",
+      description: "The path of the template to use for code generation."
+    )
 
-private let paramsOption = VariadicOption<String>(
-  "param",
-  default: [],
-  description: "List of template parameters"
-)
+    static let params = VariadicOption<String>(
+      "param",
+      default: [],
+      description: "List of template parameters"
+    )
+  }
+}
 
 extension ParserCLI {
   private var filterOption: Option<String> {
```

Comment:
I was going to ask why this wasn't above with the other options, but after looking at the rest of the code, it can't 🤦‍♂️ 
Unless we make it a `func` that accepts a `parserType`?

Reply:
Yeah, I actually tried that at first then realised. I'll play with the alternative of having a `func …(… : ParserType)` to see how it looks